Sources/NetworkProtection/Diagnostics/NetworkProtectionError.swift

```diff
@@ -142,6 +146,8 @@ public enum NetworkProtectionError: LocalizedError, CustomNSError {
                 .wireGuardDnsResolution,
                 .startWireGuardBackend,
                 .noAuthTokenFound,
+                .keychainError,
```

Comment:
Are we purposely hiding the underlying error for this one?  It could be nice to get it in pixels.

Reply:
Not at all - in fact I posted a comment to one of these PRs saying that I'm gonna update this to include the Error associated value, but I don't see it on any of them now. 🤔